getsolus/packages
# lime3ds: Update to 2118

**Summary**

Bugfixes
- Fix hotkeys not working correctly after accessing settings
- Fix Vulkan renderer causing crashes on devices that don't support Vulkan 1.3

Enhancements
- Combine `lime3ds-cli` binary into the GUI binary, and remove suffixes. You can use `lime3ds` for either purpose now.
- Internal rebrand to replace Citra references

**Test Plan**

- Open lime3ds
- Launch a title
- Test inputs, touch, hotkeys, etc.
- Make sure graphics look like they should

**Checklist**
